src/main/java/lotto/domain/rating/Rating.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain.rating;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum Rating {
+    FIRST(6, 2000000000),
```

Comment:
https://countryxide.tistory.com/52

```suggestion
    FIRST(6, 2_000_000_000),
```